src/instruments/src/EFB/Dispatch/Pages/FuelPage.tsx

```diff
@@ -209,6 +211,25 @@ export const FuelPage = () => {
         }
     };
 
+    const refuelButtonStatus = () => {
+        if (simGroundSpeed > 0.1 || eng1Running || eng2Running || !isOnGround || (!busDC2 && !busDCHot1)) {
```

Comment:
This condition is already present above. We should store it in a variable instead.

Reply:
Done.